### ISSUE TITLE
Ordering studies

### ISSUE THREAD

**Issue by nclaidiere** [open-cogsci/omm-server#122]
In the list of studies the order is based on last on top, could we make it customisable? With say a list with numbers for instance? That would give us more flexibility.

![ordering_studies](https://user-images.githubusercontent.com/50901388/163544867-bc7eda9e-6c3d-4ed8-b315-aec24015b614.PNG)

**Comment by smathot** [comment]
This would be nice, but I'm marking this as won't fix for now since it's additional functionality rather than a bug fix.

**Comment by dschreij** [comment]
My intention was to implement a search bar, but I never got around to it. Or actually I tried, but I couldn't fit the search bar in the UI nicely and then left it 😓. What ways would you like to be able to sort the studies like? Just randomly, or order them by title, creation date, etc. etc. Random sort is difficult to achieve and also requires database changes (because the sort order needs to be stored). If you just want to sort by properties of the study, I may be able to look what is possible at some point.

**Comment by dschreij** [comment]
Yeah, that is difficult to achieve. What would be the purpose of this? Get the most important or active studies at the top? Wouldn't archiving old stories also declutter you current studies list? And would adding a search bar alleviate this problem a bit?
